app/builders/messages/instagram/message_builder.rb

```diff
@@ -69,9 +69,24 @@ def contact
   end
 
   def conversation
```

Comment:
same comments as per facebook message builder. 

lets break it down to smaller methods which are readable.